src/utils/swapMinAmounts.ts

```diff
@@ -1,18 +1,37 @@
-import { asNumber, asString } from 'cleaners'
+import { asNumber, asObject, asString } from 'cleaners'
 import {
   EdgeAccount,
   EdgeCurrencyWallet,
   EdgePluginMap,
   EdgeSwapRequest,
   EdgeSwapRequestOptions
 } from 'edge-core-js'
+import fetch from 'node-fetch'
+
+import { config } from './config'
 
 interface SwapReqInfo {
   fromWallet: EdgeCurrencyWallet
   toWallet: EdgeCurrencyWallet
   amount: number
 }
 
+interface SingleCurrencyPairMinAmt {
+  [pluginName: string]: number
+}
+
+const baseUri: string = config.ratesServerAddress
+const route: string = 'v1/exchangeRate/'
+const queryStr: string = '?currency_pair=USD_'
```

Comment:
All of this don't really need to be typed.

Reply:
I was able to remove typing for `route` and `queryStr`. TypeScript did not like it when I removed the typing for `baseUri`, so I will be keeping that as is.